Course_Part_04/src/Assignment15_IsVowel.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Arrays;
+
+public class Assignment15_IsVowel {
+
+    public static void main(String[] args) {
+
+        String[] letters = {"a", "b", "e", "P", "O", "l", "K"};
+
+        for (String letter : letters) {
+            if (isVowel(letter)) {
+                System.out.printf("The char %s is a vowel%n", letter);
+            } else {
+                System.out.printf("The char %s is a consonant%n", letter);
+            }
+        }
+
+    }
+
+    private static boolean isVowel(String let) {
```

Comment:
`let` doet mij denken aan javascript ;-) .. een betere naam zou kunnen zijn letter of character (char kan niet wat dat is een gereserveerd keyword).